internal/tiltfile/files.go

```diff
@@ -339,7 +339,22 @@ func (s *tiltfileState) helm(thread *starlark.Thread, fn *starlark.Builtin, args
 		return nil, fmt.Errorf("Argument 0 (path): %v", err)
 	}
 
-	yaml, err := s.execLocalCmdArgv("helm", "template", localPath.path)
+	templates, err := ioutil.ReadDir(filepath.Join(localPath.path, "templates"))
+	if err != nil {
+		return nil, fmt.Errorf("Error reading chart template directory at %s: %v", localPath.path, err)
```

Comment:
i feel like this error message could be a lot better?

Before this change, the error message would come from Helm and look like "no Chart.yaml exists in directory "/home/nick/src/tilt.build/src"", but not you're short-circuiting that